spyder/widgets/browser.py

```diff
@@ -52,28 +84,154 @@ def acceptNavigationRequest(self, url, navigation_type, isMainFrame):
+            name=WebViewActions.Copy,
+            text=_("Copy"),
+            # icon=self.create_icon(''),
+            triggered=lambda: copy_action.trigger(),
+            context=Qt.WidgetWithChildrenShortcut,
+        )
+
+        self.zoom_in_action = self.create_action(
+            name=WebViewActions.ZoomIn,
+            text=_("Zoom in"),
+            icon=self.create_icon('zoom_in'),
+            triggered=self.zoom_in,
+            context=Qt.WidgetWithChildrenShortcut,
+        )
+        self.zoom_out_action = self.create_action(
+            name=WebViewActions.ZoomOut,
+            text=_("Zoom out"),
+            icon=self.create_icon('zoom_out'),
+            triggered=self.zoom_out,
+            context=Qt.WidgetWithChildrenShortcut,
+        )
+
+        original_inspect_action = self.pageAction(
+            QWebEnginePage.InspectElement)
+        inspect_action = self.create_action(
+            name=WebViewActions.Inspect,
+            text=_("Inspect"),
+            # icon=self.create_icon(''),
+            triggered=lambda: original_inspect_action.trigger(),
+            context=Qt.WidgetWithChildrenShortcut,
+        )
+
+        original_refresh_action = self.pageAction(QWebEnginePage.Reload)
+        self.create_action(
+            name=WebViewActions.Refresh,
+            text=_("Refresh"),
+            icon=self.create_icon('refresh'),
+            triggered=lambda: original_refresh_action.trigger(),
+            context=Qt.WidgetWithChildrenShortcut,
+        )
+        original_stop_action = self.pageAction(QWebEnginePage.Stop)
+        self.create_action(
+            name=WebViewActions.Stop,
+            text=_("Stop"),
+            icon=self.create_icon('stop'),
+            triggered=lambda: original_stop_action.trigger(),
+            context=Qt.WidgetWithChildrenShortcut,
+        )
+
+        menu = self.create_menu(WebViewMenus.Context)
+        self.context_menu = menu
+        self.add_item_to_menu(
+            back_action,
+            menu=menu,
+            section=WebViewMenuSections.Move,
+        )
+        self.add_item_to_menu(
+            forward_action,
+            menu=menu,
+            section=WebViewMenuSections.Move,
+        )
+        self.add_item_to_menu(
+            select_all_action,
+            menu=menu,
+            section=WebViewMenuSections.Select,
+        )
+        self.add_item_to_menu(
+            copy_action,
+            menu=menu,
+            section=WebViewMenuSections.Select,
+        )
+        self.add_item_to_menu(
+            self.zoom_in_action,
+            menu=menu,
+            section=WebViewMenuSections.Zoom,
+        )
+        self.add_item_to_menu(
+            self.zoom_out_action,
+            menu=menu,
+            section=WebViewMenuSections.Zoom,
+        )
+        self.add_item_to_menu(
+            inspect_action,
+            menu=menu,
+            section=WebViewMenuSections.Extras,
+        )

```

Comment:
Please use a `for` to simplify this code.

Reply:
Fixed (but it needs several for cycles, since the items are being added to different sections)